keyboards/ymd75/rev3/config.h

```diff
@@ -4,6 +4,8 @@
 #define MATRIX_COLS 9
 #define MATRIX_ROW_PINS { B7, B3, B2, B1, B0, E6, F0, F1, F4, F5, F6, F7 }
 #define MATRIX_COL_PINS { D0, D1, D2, D3, D5, D4, D6, D7, B4 }
+#undef BACKLIGHT_PIN // undefine the p
+#define BACKLIGHT_PIN B6 // backlight pin has since changed in Rev 3
```

Comment:
It would probably be better to move the original define out of the top level config.h, so that you don't have to undef here.

Reply:
That's true, but then I didn't want to change the `config.h` in the keyboard/ymd75 directory to avoid any problems with rev 1 and rev 2 boards.

Reply:
It won't, if you then add it back to the rev1 and rev2 config.h.

Reply:
Well, that's kind of the point why I didn't do that because I can't verify that my changes to config.h in rev 1 and rev 2 would work. But I suppose it shouldn't break it; it's just one line anyways.

Alright, I'm going to submit another commit.

Reply:
I did it @fauxpark